.env

```diff
@@ -0,0 +1 @@
+REACT_APP_API_KEY=6a3affea0dc3f2287d470f99deb8e2f2
```

Comment:
never commit the .env file. Always include it in the .gitignore file and make sure to never commit and/or push the .env!!!
Please revise https://bootcamp.rocketacademy.co/2-full-stack/2.2-advanced-react/2.2.7-environmental-variables